agent/src/main/java/com/appland/appmap/util/ClassUtil.java

```diff
@@ -130,4 +131,14 @@ public static Enum<?> enumValueOf(ClassLoader cl, String valueName, String... cl
   public static EnumSet<?> enumSetOf(Class<?> enumClass, String valueName) {
     return EnumSet.of((Enum) enumValueOf(enumClass.asSubclass(Enum.class), valueName));
   }
+
+  public static Method findMethod(StackTraceElement ste) {
+    try {
+      Class<?> cls = safeClassForName(Thread.currentThread().getContextClassLoader(), ste.getClassName());
```

Comment:
Yes. I'll update `safeClassForName` so it handles a `null` `ClassLoader`. I think we should do it that way, rather than having a one-argument version of `safeClassForName`, though. We've had issues in the past when the default class loader was used inappropriately. I really want the caller of `safeClassForName` to be deliberate in choosing the class loader to use.